lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1804,8 +1807,6 @@ class DxilRDATWriter : public DxilPartWriter {
         info.ShaderKind = shaderKind;
         if (pInfo2)
           pInfo2->RawShaderRef = shaderInfo;
-        if (pInfo3)
-          pInfo3->Node = nodeInfo;
```

Comment:
Since other shader info is mutually exclusive with node shader info, it made sense to integrate node and the other shader info types into the same union in info2.  Then there's no need for any info3 version.  This is a modification from the original plan that I realized made even more sense once I was modifying it.